pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -1205,6 +1217,21 @@ private Path dataDir() {
     }
   }
 
+  private Path pluginsDir() {
+    if (isFullInstantiation()) {
```

Comment:
Same as previous comment. App should not behave differently when run inside docker.

Reply:
This is a concern outside the scope of this PR.  I am being consistent with the current architecture.